modules/core/src/main/java/org/apache/ignite/internal/processors/performancestatistics/OperationType.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.performancestatistics;
+
+/**
+ * Operation type.
+ */
+public enum OperationType {
+    /** Cache operation. */
+    CACHE_OPERATION,
```

Comment:
Why we distinguish `CacheOperation` and `OperationType`?
It seems to me that this two enums can be combiner into one pretty natural.

Reply:
IMHO, it'll be strange to provide method to read cache operation with **not-only-cache-operation** type:
`void cacheOperation(UUID nodeId, OperationType type, int cacheId, long startTime, long duration);`